Makefile.devel

```diff
@@ -58,8 +58,17 @@ always-build-chplspell-venv: FORCE
 	$(MAKE) chplspell-venv; \
 	fi
 
-test: FORCE
-	cd test && start_test
+test: spectests FORCE
+	cd examples && start_test
```

Comment:
This looks like an unrelated change and I don't see it called out in the PR message

Reply:
I'll add something to the PR message along these lines:

While there, updated `make test` to run the tests in examples/ because what it had would run all of the tests (which takes something like a day). Note that `make test` is only available in source checkouts and not included in releases.

Reply:
Awesome, thanks!